### PR TITLE
ci: add support for canaries

### PR DESCRIPTION
**Pull request by kazo0**

### CODE REVIEW

**Review by jeromelaban — APPROVED** [review]

**Review by Xiaoy312 — APPROVED** [review]

**Comment by kazo0** [comment]
@jeromelaban looks like we'd need new .mobileprovision files for these new Bundle IDs. I changed them to differentiate between winui and non-winui as well as the new canary bundle ids. I don't think I have permissions to create new certs/provisioning profiles

**Comment by kazo0** [comment]
@jeromelaban this would be for the new Toolkit Canary apps though, I can re-use the ones that are for Chefs? I thought they'd be tied to a specific bundle id

**Review by agneszitte — APPROVED** [review]

**Comment by agneszitte** [comment]
> Ah that's toolkit, yes I'll fix that

**Comment by agneszitte** [comment]
> Ah that's toolkit, yes I'll fix that :)

@kazo0 you should be able to use:
image

File: samples/Uno.Toolkit.WinUI.Samples/Uno.Toolkit.WinUI.Samples.macOS/Info.plist [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	<string>10.11</string>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
-	<key>CFBundleInfoDictionaryVersion</key>
+<key>CFBundleInfoDictionaryVersion</key>
```
Comment (agneszitte) [inline samples/Uno.Toolkit.WinUI.Samples/Uno.Toolkit.WinUI.Samples.macOS/Info.plist:17]:
```suggestion
	<key>CFBundleInfoDictionaryVersion</key>
```

**Pull request merged**